roles/openshift_health_checker/action_plugins/openshift_health_check.py

```diff
@@ -134,3 +133,14 @@ def resolve_checks(names, all_checks):
         resolved.update(tag_to_checks[tag])
 
     return resolved
+
+
+def normalize(checks):
+    """Return a clean list of check names.
+
+    The input may be a comma-separated string or a sequence. Leading and
+    trailing whitespace characters are removed. Empty items are discarded.
+    """
+    if isinstance(checks, string_types):
+        checks = checks.split(',')
+    return [name.strip() for name in checks if name.strip()]
```

Comment:
Ah, this made more sense in the context of the adhoc playbook.

`ansible-playbook ... -e openshift_checks=foo,bar,baz` passes in a string.

When working on it, I thought it would be useful to handle YAML/JSON as well, because we can write `-e {"openshift_checks": ["foo","bar","baz"]}` -- not something I would do if I can write a comma-sep string... but this becomes useful if we write it to a file, like `vars/main.yml`:

```yaml
# vars/main.yml
---
openshift_checks:
  - foo
  - bar
  - baz
```

The user needs to try really hard to pass in a list of numbers...